lib/kubeclient/common.rb

```diff
@@ -202,6 +205,14 @@ def resource_name(entity_type)
         entity_type.pluralize.downcase
       end
 
+      def proxy_url_service(namespace, service, protocol)
```

Comment:
This should be:

    proxy_url_service(namespace, service, port_name)

@jimmidyson the port/protocol names are irrelevant because you address them with a name, which in @oschreib example was in the format of `port-protocol`, but in general it could be anything "foobar", etc.

As far as I saw you're required to specify the `port_name` also when the port is only one.